README.md

```diff
@@ -12,17 +12,16 @@ Minimal supported `Gradle` version: `6.6`.
 ## Table of contents
 - [Features](#features)
 - [Quickstart](#quickstart)
-  - [Apply plugin to a single-project build](#apply-plugin-to-a-single-project-build)
```

Comment:
Unfortunately not. In the #144, it is proposed to separate plugins that will be applied to one project and to several projects at once (aggregated). In this PR, this is still done by a single plugin.

The separation can be done now as part of this release (but another PR), so that the new API will assumes the presence of two plugins, or in future releases.